.github/dependabot.yaml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
shall we keep it weekly? 😄 it keeps creating pull request

Reply:
Yeah just want to try it first to see how it will works